accord-core/src/test/java/accord/impl/list/ListAgent.java

```diff
@@ -58,6 +58,7 @@ public void onInconsistentTimestamp(Command command, Timestamp prev, Timestamp n
     @Override
     public void onUncaughtException(Throwable t)
     {
+        if (t instanceof accord.coordinate.Timeout) return;
```

Comment:
I don't like this but SPL sees it and now calls agent so had to do else we fail nearly every time... I feel like this is calling out that SPL should have different logic which would allow us to remove this